### ISSUE TITLE
Download Link

### ISSUE THREAD

**Issue by drive28** [fwang91/IMDb-Face#4]
Hi,

Is the link still active ? When I go to the link I am unable to see the files. Is it possible for you to give a dropbox link ?

Thanks,

**Comment by fwang91** [comment]
Yes, the link is still active. We have provided the google drive link, you can try it.

**Comment by bach-zouk** [comment]
I cant download the csv file via google drive. link doesnt exist
